main.go

```diff
@@ -10,40 +11,40 @@ import (
 
 func main() {
 	// Needs as argument a path to an Antora project
-	if len(os.Args) < 2 {
-		fmt.Println("Please provide a path to an Antora project")
-		os.Exit(1)
-	}
-	path := os.Args[1]
+	var antoraPath string
+	flag.StringVar(&antoraPath, "antoraPath", "/docs", "Path to Antora document sources")
 
-	// Remove the trailing slash
-	if strings.HasSuffix(path, "/") {
-		path = strings.TrimSuffix(path, "/")
-	}
+	// Module to check. If none passed, it is assumed to be ROOT
+	var module string
+	flag.StringVar(&module, "module", "ROOT", "Module to analyze")
+
+	// File to parse and analyze; if none is passed, it is assumed to
+	// be a standard Antora navigation file at the ROOT module.
+	var filename string
+	flag.StringVar(&filename, "filename", "/modules/ROOT/nav.adoc", "File to analyze")
+
+	flag.Parse()
 
 	// Main block
-	errors := check(path)
+	errors := check(antoraPath, module, filename)
 
 	// If there are errors, print the list of orphan files
 	if len(errors) > 0 {
 		for _, file := range errors {
-			fmt.Println(fmt.Sprintf("File '%s' not in 'nav.adoc'", file))
+			fmt.Fprintf(os.Stdout, "File '%s' not in '%s'\n", file, filename)
 		}
 		os.Exit(1)
 	}
 
 	// No errors, all good!
-	fmt.Println("No orphan files in modules/ROOT/nav.adoc")
-	os.Exit(0)
-
 }
 
 // Takes a path and assumes it's a valid Antora project. It lists all pages
-// contained in the ROOT module and then walks the file system to verify that
-// all relevant files are referenced in the `nav.adoc` file.
-func check(path string) []string {
+// contained in the module and then walks the file system to verify that
+// all relevant files are referenced in the file.
+func check(antoraPath string, module string, filename string) []string {
 	// We assume that the project follows a standard Antora layout
-	startPath := path + "/modules/ROOT/pages"
+	startPath := antoraPath + "/modules/" + module + "/pages"
```

Comment:
```suggestion
	startPath := filepath.Join(antoraPath, "modules", module, "pages")
```

Again, please use `filepath.Join` when concatenate path fragments. It actually takes an arbitrary number of arguments.